clients/cli/nmcli.c

```diff
@@ -51,7 +51,6 @@
 # define NMCLI_VERSION VERSION
```

Comment:
double "it" in the commit msg... drop a useless 'it' in "drop a useless comment" ;-)